src/Restaurant.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
+import Order from "./Order.js";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
-    <div>
+    <div className="restaurant">
       <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+      <ul className="ul-styling">
+        <Order orderType="Pizzas" />
```

Comment:
Good stuff here @FarhanaFKhan - think of another way you could also render each of the orders without rendering them each individually one at a time.